src/Resources/views/styleguide/design-elements/typography.html.twig

```diff
@@ -3,6 +3,13 @@
 
 
 {% block component_list %}
+    <section class="c-component--typography  s-cms-content  s-cms-content--introduction">
+        <p>
+            Is this my espresso machine? Wh-what is-h-how did you get my espresso machine? You know what? It is beets.
+            I've crashed into a beet truck. My dad once told me, laugh and the world laughs with you, Cry, and I'll give
+            you something to cry about you little bastard!
+        </p>
+    </section>
     <section class="c-component--typography  s-cms-content" style="max-width: 980px;">
 
         <h1>The quick brown fox jumps over the lazy dog.</h1>
```

Comment:
We zouden de H1 weg kunnen laten? Die kun je eigenlijk nooit via het CMS toevoegen

Reply:
Ja eens, wat mij betreft hoort ie ook niet thuis in de tinymce.

Reply:
Er moet wel ergens een `<h1>` op de typografie pagina komen te staan toch? Want je maakt nooit een "pagina titel" element ergens anders in de styleguide.

Reply:
Eh ja, ik zou er weer een h1 neer kunnen zetten net zoals op alle andere styleguide pagina's. Maar dan ook met dezelfde styling als op die andere pagina's. Nu ontstond er de verwarring of er ook een h1 in de tinymce moest komen en dat is bijna nooit aan de orde. Ik zal het zodra ik een momentje heb wel aanpassen.

Reply:
@7ochem ik heb dezelfde `<h1>` toegevoegd als op de component templates.

Reply:
Oh dat bedoelde ik niet, haha. Ik bedoelde meer dat er nu nergens in de styleguide is te zien hoe de `<h1>` in een project eruit ziet (zoals in de titel van pagina's van een website)

Reply:
Ah juist, ik gebruik zelf dat typografie template vooral om te kijken hoe tekst er uit komt te zien als het om tekst gaat die uit de tinymce editor komt. En een pagina titel is vaak een wat groter vormgegeven object dat niet zo makkelijk past op een styleguide template. Vandaar dat ie er nu uit is.